README.md

```diff
@@ -38,17 +38,24 @@ observation. The following fields are required:
 
 The object will be returned, with the fields `id` and `timestamp` set.
 
+The property `ref` can also be optionally set, to indicate that the observation *observes* the OSM element with ID `ref`.
+
 #### `GET /observations/:id`
 
 Fetch an observation by its `id`. An array of JSON objects will be returned.
 Usually there will only be one result, but in forking situations (e.g. two
 devices create offline edits of the same observation then sync) there can be
 multiple results.
 
-#### `PUT /observations/:id`
+#### `PUT /observations/:version`
```

Comment:
@noffle, I'm not suggesting only updating the latest timestamped head. I'm suggesting updating based on the versionId of the observation that you put to this endpoint. I'm saying that the PUT url should be the `:id` but the observation that is updated is the one that is in the body of the request (i.e. use the versionId of the obs in the request). This is what is currently done in both the normal OSM API and in our osm-p2p api. I think the URL should be `:id` because you are updating the observation at that id, you aren't updating a version, because you can't. The client will have a version of the observation, and when the client PUTs an update, it should update the version that the client had, so we should use the versionId submitted by the client.